src/components/common.js

```diff
@@ -26,6 +26,22 @@ export function validateProps(props : Object, required : boolean = true) {
         }
     }
 
+    // Prop Validations for the credit button
+
+    // Tiny size for credit button is not supported
+    if (props.style.label && props.style.size) {
```

Comment:
Can we make sure there's a null check for `props.style`?

We can probably make this whole line:

`if (props.style && props.style.label === 'credit' && props.style.size === 'tiny') {`